dataservices/dataservices.go

```diff
@@ -175,6 +176,30 @@ func process(iter store.Iterator) data.Dataset {
 	return all
 }
 
+func buildQuery(params url.Values) store.Query {
+
+	types := strings.Split(params.Get("type"), ",")
+	subTypes := strings.Split(params.Get("subType"), ",")
```

Comment:
As noted in previous PR, use `type`, `subType`, `startDate` and `endDate` for variable names.